--- origsrc/polymer-0.3.1/style/polymer.h	2005-05-14 12:51:37.000000000 -0500
+++ src/polymer-0.3.1/style/polymer.h	2008-07-28 20:00:16.750000000 -0500
@@ -257,6 +257,9 @@
     
     bool eventFilter(QObject *, QEvent *);
 
+    inline QColor getColor(const QColorGroup &cg, const ColorType t, const bool enabled = true)const;
+    inline QColor getColor(const QColorGroup &cg, const ColorType t, const WidgetState s)const;
+
     QWidget* hoverWidget;
 protected slots:
     void khtmlWidgetDestroyed(QObject* w);
@@ -264,9 +267,6 @@
     //Animation slots.
     void updateProgressPos();
     void progressBarDestroyed(QObject* bar);
-
-    inline QColor getColor(const QColorGroup &cg, const ColorType t, const bool enabled = true)const;
-    inline QColor getColor(const QColorGroup &cg, const ColorType t, const WidgetState s)const;
 private:
 // Disable copy constructor and = operator
     PolymerStyle( const PolymerStyle & );
--- origsrc/polymer-0.3.1/style/polymer.pro	2005-05-14 12:51:37.000000000 -0500
+++ src/polymer-0.3.1/style/polymer.pro	2008-07-28 20:01:43.562500000 -0500
@@ -5,8 +5,9 @@
 TEMPLATE = lib
 INCLUDEPATH += .
 
-CONFIG = qt warn_on release thread
+CONFIG = qt warn_on release thread plugin
 
 # Input
 HEADERS += kcpuinfo.h kstyle.h misc.h polymer.h
 SOURCES += kcpuinfo.cpp kstyle.cpp misc.cpp polymer.cpp
+LIBS += -lXrender -lX11
